pom.xml

```diff
@@ -0,0 +1,336 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0                              http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
In dieser Zeile scheint die Code Formatierung nicht richtig funktioniert zu haben.

Reply:
Idea hats nicht korrigiert.